sda-download/dev_utils/compose.yml

```diff
@@ -105,6 +105,22 @@ services:
       - "8443:8443"
     restart: always
 
+  reencrypt:
+    image: ghcr.io/neicnordic/sensitive-data-archive:v0.3.7
+    command: [ sda-reencrypt ]
+    container_name: reencrypt
+    depends_on:
+      certfixer:
+        condition: service_completed_successfully
+    ports:
+      - "50051:50051"
+    restart: always
+    volumes:
+      - ./config.yaml:/config.yaml
+      - ./:/dev_utils/
+      - ./iss.json:/iss.json
+      - certs:/dev_utils/certs
```

Comment:
Shouldn't this service also use certificates?

Reply:
I tried with certificates but failed many times (as I mentioned in the PR description). I discussed with @dbampalikis and think we may probably skip it for this PR and to get the basic functionality ready first. And then add the TLS feature later on. But if you have some quick suggestions, I can give it another try 😄

Reply:
mTLS is enabled in the commit 9f549c32